chainercv/transforms/bbox/rotate_bbox.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+
+
+def rotate_bbox(bbox, angle, size):
+    """Rotate bounding boxes by degrees.
+
+    Args:
+        bbox (~numpy.ndarray): An array whose shape is :math:`(R, 4)`.
+            :math:`R` is the number of bounding boxes.
+        angle (float): Clock-wise rotation angle (degree) in [-180, 180].
```

Comment:
Isn't this counter clock-wise?

(rotated by angle 10)
![figure_1](https://user-images.githubusercontent.com/2062128/45663704-d45e8500-bb42-11e8-9508-32153ef67aae.png)

Reply:
yes, it is counter clock-wise!
thank you.